### PR TITLE
i2pd: add vlogger

### PR DESCRIPTION
**Pull request by steinex**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

File: srcpkgs/i2pd/files/i2pd/log/run [hunk 1]
```diff
@@ -0,0 +1 @@
+/usr/sbin/vlogger
```
Comment (Piraty) [inline srcpkgs/i2pd/files/i2pd/log/run:1]: please align with how other `log/run` use `vlogger`

Comment (steinex) [reply srcpkgs/i2pd/files/i2pd/log/run:1]: It's a symlink, I thought thats how it's supposed to be these days? Did I miss something?

Comment (Piraty) [reply srcpkgs/i2pd/files/i2pd/log/run:1]: oh i missed it's a symlink.
`fd --full-path 'log/run' -l ` reveals 25% of them are just symlinks to vlogger,  all others are scripts that `exec vlogger`

Comment (classabbyamp) [reply srcpkgs/i2pd/files/i2pd/log/run:1]: symlink to /usr/bin/ not /usr/sbin

Comment (steinex) [reply srcpkgs/i2pd/files/i2pd/log/run]: I think we have it now. I'm sorry for the confusion. ;)

**Pull request merged**